script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -287,7 +286,7 @@ I can't say this out loud, but her smile really isn't vampire-like at all.
 [sha:360576089921dcaa81eb0a76e139a3282e9ce29a]{
 -- Page 314, Offset 25053.
 -- ……それはそれとして、今までそんなに怒鳴り散らしてたかな、俺……。
-...Have I really been yelling at her that often?
+That said... have I really been yelling at her that often?
```

Comment:
Not sure I like the 'that said', this thought doesn't really contradict the previous

Reply:
I think the change is fine. It's more like he got distracted by her cuteness and then put himself back on track within his thoughts.

Reply:
I agree with Yusuke.